src/Uno.UWP/ApplicationModel/Contacts/ContactQueryDesiredFields.cs

```diff
@@ -6,23 +6,23 @@ namespace Windows.ApplicationModel.Contacts
 	/// Defines which fields must exist on a contact in order to match a search operation.
 	/// </summary>
 	[Flags]
-	public enum ContactQueryDesiredFields
+	public enum ContactQueryDesiredFields : uint

```

Comment:
Why are you converting `enum`s to use `uint`?

Reply:
Because they should be uints :)

Reply:
![image](https://user-images.githubusercontent.com/33458222/122591026-5cde1d00-d030-11eb-8082-ac4e177eee98.png)